tensorflow_probability/python/bijectors/conditional_bijector_test.py

```diff
@@ -48,6 +48,32 @@ def _forward_log_det_jacobian(self, _, arg1, arg2):
     raise ValueError("forward_log_det_jacobian", arg1, arg2)
 
 
+class _TestPassthroughBijector(_TestBijector):
```

Comment:
I understand you're trying to reproduce the test case from your bug report, but it feels like it would be a lot simpler to just directly test the nested dicts by passing dict args to _TestBijector. What do you think?

Reply:
I added simpler `testNestedCondition` test (see https://github.com/tensorflow/probability/pull/254/commits/462a231df8fdaef093b44b2af1848237d91eadcd). Should I remove the explicit conditional chain test or do you want me to keep it?

Reply:
Let's remove it if you don't mind, I think testNestedCondition is a little more clear and covers what we want to test.